### ISSUE TITLE
Use charts for rule metrics

### ISSUE THREAD

**Issue by keithCuniah** [recognai/rubrix#1996]
refs #1767 

![image](https://user-images.githubusercontent.com/88380932/205116435-ce1b612a-02ed-489f-8779-61e9aea7559c.png)

I have an improvement suggestion for showing data in rule metrics.
Maybe add some charts instead of raw data for rule metrics could make things more user friendly.

I would like to suggest [d3.js](https://d3js.org/) library which is an optimized library for datavisualisation in general.

**Comment by leiyre** [comment]
I'm agree that it would be nice replace in some places the textual info with more visual info. We already have vega-lite (vega-litehttps://vega.github.io/vega-lite/), it is only used for the score range. I would only work with a single data visualization library (vega o d3), personally I prefer d3 for our case.

**Comment by frascuchon** [comment]
Yes. It males sense. Even more once we will start including dataset related metrics in UI.

**Comment by dvsrepo** [comment]
Yes, it's a nice suggestion @keithCuniah !

As we have several things in the roadmap related to data visualization, let's do a brainstorming session, and define some requirements. Then you can do some research on the most suitable library for us.
